### PR TITLE
[3860] Dotfiles Update for Repositories: Version 10.0

### PR DESCRIPTION
**Pull request by liuhehe1995**
[3860](https://www.quartile.co/web#id=3860&menu_id=517&cids=3&model=project.task&view_type=form)

### CODE REVIEW

**Comment by liuhehe1995** [comment]
This PR came up with a problem I couldn't solve and have asked Lin for help.

**Comment by liuhehe1995** [comment]
For Pre－commit failed,
`pre-commit run --all-files`  is successful in my local env.
image
image

**Comment by liuhehe1995** [comment]
@kanda999 さん
For OCB failed, Lin's advice is add website_product_stock module in test-requirements.txt
But website_product_stock is exist in pci-private.
Now we've come up with 2 solutions:
1,Move website_product_stock module from private to custom 
2,Split the module
Lin told me to ask you what to do.
Can you tell me what I'm supposed to do?

**Comment by AungKoKoLin1997** [comment]
I take over this PR.

**Comment by AungKoKoLin1997** [comment]
The pre-commit is run successfully in our local env but I plan to take care the tests CI failure because it is related with module dependency and cant finish in short time.

**Review by AungKoKoLin1997 — APPROVED** [review]

File: fix_product_attribute_line_name_search/models/product_attribute_line_hook_name_search.py
Comment (yostashiro) [inline fix_product_attribute_line_name_search/models/product_attribute_line_hook_name_search.py:1]: Why an addition of this entire file in this PR?

File: mrp_bom_component_menu/__manifest__.py
Comment (yostashiro) [inline mrp_bom_component_menu/__manifest__.py:1]: Why an addition of this entire file in this PR?

**Comment by AungKoKoLin1997** [comment]
Superseded by https://github.com/qrtl/pci-custom/pull/218.

**Pull request closed without merging**